app/src/main/res/values/strings.xml

```diff
@@ -7,7 +7,10 @@
     <string name="set_group_title">Set</string>
 
     <string name="new_group">New Group +</string>
+    <string name="invite">invite</string>
```

Comment:
I would still capitalize this. My logic: we may need a capitalized string, but we'll never need an all-lowercase string